### DISCUSSION TITLE
[Training Log 2022-07-12] Come in to solve a puzzle😉

### DISCUSSION THREAD

**Discussion by zh-zheng** [OpenBMB/CPM-Live#61] [Training Logs 训练日志]
# CPM-Live Training Log (July, 12)

Time: July, 12 2022 16:00

Recorder: @zh-zheng

## Loss
- Begin: 2.53
- End: 2.54
	
## Completed Data
- $\approx$ 565.26GB

## Average Grad Norm
- 0.95

## Progress
- 60.69%

## Comment

The training has entered a stable period. Since our model is capable of doing both generation and understanding tasks, today we test it on a Chinese poetry comprehension task.

Explanation: 沿着松柏小径直奔神灵宫

Option 👍: 松柏一径趋灵宫

Option 😄: 转入玄宫松柏间

Option 🎉: 竹柏深中一径开

Opiton 🚀: 杖藜松径叩莲宫

Which one do you think is correct? Vote for it with the emoji! We will post CPM-Ant's answer tomorrow.

**Comment by Kunlun-Zhu** [comment]
My guess is the first one.
